### ISSUE TITLE
Avoidable confusion between decimal (k, M, G...) and binary (K, M[i], G[i]...) prefixes

### ISSUE THREAD

**Issue by sleinen** [cnp3/ebook#51]
In "Connecting two hosts", one of the very first sections, it is correctly pointed out that bit rates are always expressed in decimal powers.  But then, the text writes "Kbps" instead of "kbps" (or kb/s)! I find this disturbing.  At least for k/K, we have a strong convention that distinguishes the decimal variant (k=1000) from the binary one (K=1024).  So I would suggest that all instances of "Kbps" be replaced with "kbps" throughout the text.  I can volunteer to propose a PR for this.

More recently, there have been attempts to introduce explicit binary versions of the higher-order prefixes, e.g.

* A MiB ("Mebibyte") is 1048576 bytes
* A GiB ("Gibibyte") is 1073741824 bytes

...and so on for Ti (Tebi-), Pi (Pebi-).  Those haven't really caught on widely, but in the interest of clarity might make sense to use when the binary variant is meant.  Especially for a textbook in networking, where decimal and binary conventions coexist to create confusion.  What do people think?

**Comment by qdeconinck** [comment]
Indeed, the distinction between the decimal version and the power of two one could be better, and I would be happy to review such a PR improving this point :-) As a personal note, I would be interested if you can provide a source about the "strong convention that distinguishes the decimal variant (k=1000) from the binary one (K=1024)".
